[2024-01-26T17:39:42.702-0300] {logging_mixin.py:151} INFO - Changing /mnt/c/Users/filipe.macedo_ifood/Documents/how-desafio-03/airflow/logs/dag_id=how-desafio-03/run_id=scheduled__2024-01-26T20:15:00+00:00/task_id=invoke_lambda_function permission to 509
[2024-01-26T17:39:42.755-0300] {logging_mixin.py:151} INFO - Changing /mnt/c/Users/filipe.macedo_ifood/Documents/how-desafio-03/airflow/logs/dag_id=how-desafio-03/run_id=scheduled__2024-01-26T20:15:00+00:00/task_id=invoke_lambda_function permission to 509
[2024-01-26T17:39:42.783-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: how-desafio-03.invoke_lambda_function scheduled__2024-01-26T20:15:00+00:00 [queued]>
[2024-01-26T17:39:42.798-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: how-desafio-03.invoke_lambda_function scheduled__2024-01-26T20:15:00+00:00 [queued]>
[2024-01-26T17:39:42.798-0300] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-26T17:39:42.825-0300] {taskinstance.py:1382} INFO - Executing <Task(LambdaInvokeFunctionOperator): invoke_lambda_function> on 2024-01-26 20:15:00+00:00
[2024-01-26T17:39:42.827-0300] {standard_task_runner.py:57} INFO - Started process 970 to run task
[2024-01-26T17:39:42.829-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'how-desafio-03', 'invoke_lambda_function', 'scheduled__2024-01-26T20:15:00+00:00', '--job-id', '267', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpx_zg669x']
[2024-01-26T17:39:42.829-0300] {standard_task_runner.py:85} INFO - Job 267: Subtask invoke_lambda_function
[2024-01-26T17:39:42.886-0300] {logging_mixin.py:151} INFO - Changing /mnt/c/Users/filipe.macedo_ifood/Documents/how-desafio-03/airflow/logs/dag_id=how-desafio-03/run_id=scheduled__2024-01-26T20:15:00+00:00/task_id=invoke_lambda_function permission to 509
[2024-01-26T17:39:42.887-0300] {task_command.py:416} INFO - Running <TaskInstance: how-desafio-03.invoke_lambda_function scheduled__2024-01-26T20:15:00+00:00 [running]> on host 14111-NB.
[2024-01-26T17:39:42.985-0300] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='how-desafio-03' AIRFLOW_CTX_TASK_ID='invoke_lambda_function' AIRFLOW_CTX_EXECUTION_DATE='2024-01-26T20:15:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-26T20:15:00+00:00'
[2024-01-26T17:39:42.985-0300] {lambda_function.py:211} INFO - Invoking AWS Lambda function: how-desafio-03 with payload: {"date": "1706301000"}
[2024-01-26T17:39:42.998-0300] {base.py:73} INFO - Using connection ID 'aws_default' for task execution.
[2024-01-26T17:39:42.998-0300] {connection_wrapper.py:340} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2024-01-26T17:39:44.088-0300] {lambda_function.py:220} INFO - Lambda response metadata: {'RequestId': 'ff7b7be7-c95a-4701-a121-1f399e26e1d4', 'HTTPStatusCode': 202, 'HTTPHeaders': {'date': 'Fri, 26 Jan 2024 20:40:07 GMT', 'content-length': '0', 'connection': 'keep-alive', 'x-amzn-requestid': 'ff7b7be7-c95a-4701-a121-1f399e26e1d4', 'x-amzn-remapped-content-length': '0', 'x-amzn-trace-id': 'root=1-65b418a7-5c83adec6138fb0b03765acd;sampled=0'}, 'RetryAttempts': 0}
[2024-01-26T17:39:44.089-0300] {lambda_function.py:230} INFO - Lambda function invocation succeeded: {'RequestId': 'ff7b7be7-c95a-4701-a121-1f399e26e1d4', 'HTTPStatusCode': 202, 'HTTPHeaders': {'date': 'Fri, 26 Jan 2024 20:40:07 GMT', 'content-length': '0', 'connection': 'keep-alive', 'x-amzn-requestid': 'ff7b7be7-c95a-4701-a121-1f399e26e1d4', 'x-amzn-remapped-content-length': '0', 'x-amzn-trace-id': 'root=1-65b418a7-5c83adec6138fb0b03765acd;sampled=0'}, 'RetryAttempts': 0}
[2024-01-26T17:39:44.158-0300] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=how-desafio-03, task_id=invoke_lambda_function, execution_date=20240126T201500, start_date=20240126T203942, end_date=20240126T203944
[2024-01-26T17:39:44.209-0300] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-26T17:39:44.267-0300] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
